Lib/test/test_cmd_line.py

```diff
@@ -972,10 +972,15 @@ def test_python_legacy_windows_fs_encoding(self):
 
     @unittest.skipUnless(support.MS_WINDOWS, 'Test only applicable on Windows')
     def test_python_legacy_windows_stdio(self):
-        code = "import sys; print(sys.stdin.encoding, sys.stdout.encoding)"
-        expected = 'cp'
-        rc, out, err = assert_python_ok('-c', code, PYTHONLEGACYWINDOWSSTDIO='1')
-        self.assertIn(expected.encode(), out)
+        code = "import sys; print(sys.stdin.encoding, sys.stdout.encoding, file=sys.stderr)"
+        env = os.environ.copy()
+        env['PYTHONLEGACYWINDOWSSTDIO'] = '1'
+        p = subprocess.Popen([sys.executable, '-c', code],
```

Comment:
```suggestion
        # Use Popen directly to ensure stdin/out are console handles
        p = subprocess.Popen([sys.executable, '-c', code],
```

I'm betting it got mass converted to `assert_python_ok`, so let's leave a comment to avoid that in the future.